hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -105,8 +105,10 @@ const (
 	HostedClusterAnnotation = "hypershift.openshift.io/cluster"
 	// HasBeenAvailableAnnotation is an implementation detail to check if HC has ever been available.
 	// This is useful for e.g. monitor duration from creation to available and avoid noise from condition flipping.
-	HasBeenAvailableAnnotation     = "hypershift.openshift.io/HasBeenAvailable"
-	clusterDeletionRequeueDuration = 5 * time.Second
+	HasBeenAvailableAnnotation          = "hypershift.openshift.io/HasBeenAvailable"
+	clusterDeletionRequeueDuration      = 5 * time.Second
+	reportingGracePeriodDuration        = 100 * time.Second
```

Comment:
Could these parameters be overwritten from a config (if needed)?

The defaults look ok, but if we perform some testing, we may want to adjust some values.